tsconfig.json

```diff
@@ -1,13 +1,12 @@
 {
   "compilerOptions": {
     "target": "es2021",
-    "module": "commonjs",
+    "module": "NodeNext",
     "declaration": true,
     "outDir": "dist",
     "strict": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "moduleResolution": "node",
```

Comment:
Why did you remove this instead of changing it to "NodeNext"?

Reply:
its implicit anyway, you cant put any other value.